mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/MarketplaceScreen.tsx

```diff
@@ -112,33 +111,39 @@ export function MarketplaceScreen (): JSX.Element {
     )
   }
 
-function ExchangeItemRow ({ image, name, url, testID, border }: ExchangeProps & { testID: string } & { border: boolean}): JSX.Element {
+function ExchangeItemRow ({ image, name, url, testID, isLast }: ExchangeProps & { testID: string } & { isLast: boolean}): JSX.Element {
     return (
-      <ThemedTouchableOpacityV2
-        onPress={async () => await openURL(url)}
-        style={tailwind('flex flex-row mx-5 py-4 items-center justify-between', { 'border-b-0.5': border })}
-        testID={testID}
+      <ThemedViewV2
+        style={tailwind('border-b-0.5', { 'border-0': isLast })}
```

Comment:
`ThemedViewV2` has no border by default
```suggestion
        style={tailwind({ 'border-b-0.5': !isLast })}
```

Reply:
updated